### DISCUSSION TITLE
iSCSI does not appear in diskpart/setup.exe when boot via EFI

### DISCUSSION THREAD

**Discussion by uzytkownik** [ipxe/ipxe#840] [General]
I'm trying to install Windows on iSCSI on system with EFI (I did successfully did it on CSM). Unfortunately I'm stuck as the disk does not appear in the WinPE (setup.exe diskpart) despite being sanhook'ed. 

```
set net/ip <IP>
set initiator-iqn <INITIATOR>
sanhook iscsi:<ISCSI TARGET>
kernel wimboot
initrd amd64/media/Boot/BCD BCD
initrd amd64/media/Boot/boot.sdi boot.sdi
initrd amd64/media/sources/boot.wim boot.wim
boot
```

iPXE shows mapping device as 0x80(?). Note that I do not control DHCP and I cannot configure iSCSI that way.

**Comment by NiKiZe** [comment]
Use latest wimboot and drop BCD and boot.sdi
```
set net/ip <IP>
set initiator-iqn <INITIATOR>
sanhook iscsi:<ISCSI TARGET>
kernel wimboot
initrd amd64/media/sources/boot.wim boot.wim
boot gui
```

Where does boot.wim come from? Does the network properly work when you enter winpe?
Does `wpeutil WaitForNetwork` run properly?
(we should assume that all this is fine if you are using the same wim as your working setup, but would still like to have it verified)
